### PR TITLE
Add bootstrap checkout icon

### PR DESCRIPTION
**Pull request by n9iels**
# The problem

When an article is checked out  by another user, you will see on the front-end a legacy icon without text
checkout-icon old
# Expected result

It should be nice if see a Bootstrap icon, or a legacy icon, with text
# How to test this patch

1) Make sure you checkout an article on the font-end by following this steps
2) Create a second user who can edit articles on front-end
3) Open two browsers, for example FireFox and Goolge Chrome
4) Login on the front-end with a user on firefox and edit an article, stay in edit mode!
5) Login on the front-end with a second user on Google Chrome and notice the same dropdown as the screenshot above
6) Applay the patch
7) Notice the following dropdown
checkout-icon new
8) Go to components/com_content/helpers/icon.php, and edit line 134 to:
`public static function edit($article, $params, $attribs = array(), $legacy = true)`
9) Refresh Google Chrome, and notice the following dropdown:
checkout-icon legacy new

### CODE REVIEW

File: components/com_content/helpers/icon.php [hunk 1]
```diff
@@ -157,12 +157,23 @@ public static function edit($article, $params, $attribs = array(), $legacy = fal
 			&& $article->checked_out != $user->get('id'))
 		{
 			$checkoutUser = JFactory::getUser($article->checked_out);
-			$button       = JHtml::_('image', 'system/checked_out.png', null, null, true);
 			$date         = JHtml::_('date', $article->checked_out_time);
 			$tooltip      = JText::_('JLIB_HTML_CHECKED_OUT') . ' :: ' . JText::sprintf('COM_CONTENT_CHECKED_OUT_BY', $checkoutUser->name)
 				. ' <br /> ' . $date;
 
-			return '<span class="hasTooltip" title="' . JHtml::tooltipText($tooltip . '', 0) . '">' . $button . '</span>';
+			if($legacy)
```
Comment (zero-24) [inline components/com_content/helpers/icon.php]: @n9iels can you add one space after `if`?

File: components/com_content/helpers/icon.php [hunk 2]
```diff
@@ -157,12 +157,23 @@ public static function edit($article, $params, $attribs = array(), $legacy = fal
...
+			{
+				$button = JHtml::_('image', 'system/checked_out.png', null, null, true);
+				$text = '<span class="hasTooltip" title="' . JHtml::tooltipText($tooltip . '', 0) . '">' . $button . '</span> ' . JText::_('JLIB_HTML_CHECKED_OUT');
```
Comment (zero-24) [inline components/com_content/helpers/icon.php]: @n9iels can you agline the `=` here with spaces?

**Comment by zero-24** [comment]
tested successful and commented on very mirror CS. Good change @n9iels <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6248">issues.joomla.org/joomla-cms/6248</a>.</sub>

**Comment by n9iels** [comment]
can you explain what you mean by: agline the `=`
Thanks for testing!

**Comment by n9iels** [comment]
ow, never mind. I understand

**Comment by zero-24** [comment]
>https://github.com/n9iels/joomla-cms/commit/6604e1e843376480882ef831613c3f8499363a9d

:+1: Thanks :smile: One more tester here!

**Comment by Erftralle** [comment]
@test: Works fine for me. Thanks.

I noticed a small difference regarding to your test instructions when reaching point 7. I saw the text **Checked out**, not **Edit** ;-) .

**Comment by n9iels** [comment]
@Erftralle thanks for testing! Yes that's right, I changed that after I submit this PR. Seems a more logic text in my opinion :-)

**Comment by Erftralle** [comment]
>Seems a more logic text in my opinion

Definetly, I agree.

**Comment by zero-24** [comment]
@n9iels sorry but again Travis is a bad boy :smiley: There is a PR that fix the last error: https://github.com/n9iels/joomla-cms/pull/1 / https://travis-ci.org/joomla/joomla-cms/jobs/52659140

As it is no functional change i move this to RTC and this can merged if the CS PR is merged. Thanks!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6248">issues.joomla.org/joomla-cms/6248</a>.</sub>

**Comment by Josh-Lewis** [comment]
Good catch. If I recall right Joomla 3.3.6 displayed the icon as a non legacy icon. Was hoping that an admin could checkin a article from the frontend. ;-)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6248">issues.joomla.org/joomla-cms/6248</a>.</sub>

**Comment by hvdmeer** [comment]
@test: Worked fine, thank you.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6248">issues.joomla.org/joomla-cms/6248</a>.</sub>

**Comment by phproberto** [comment]
Merged. Thanks!

**Pull request closed without merging**